crates/bevy_diagnostic/src/system_information_diagnostics_plugin.rs

```diff
@@ -46,10 +46,15 @@ impl SystemInformationDiagnosticsPlugin {
 /// [`SystemInformationDiagnosticsPlugin`] for more information.
 #[derive(Debug, Resource)]
 pub struct SystemInfo {
+    /// OS name and version.
     pub os: String,
+    /// System kernel version.
     pub kernel: String,
+    /// CPU brand, for example 'Apple M1 Max'.
```

Comment:
I assume that's a real-world example? i.e. this string looks *exactly* like that on a real M1 Max?

Reply:
Yes. I wasn't sure what this was so ran it on my laptop and copied the output.